src/NServiceBus.AcceptanceTests/Recoverability/Retries/When_message_is_moved_to_error_queue_using_dtc.cs

```diff
@@ -0,0 +1,102 @@
+﻿namespace NServiceBus.AcceptanceTests.Recoverability.Retries
+{
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using System.Transactions;
+    using AcceptanceTesting;
+    using EndpointTemplates;
+    using Features;
+    using NUnit.Framework;
+    using ScenarioDescriptors;
+
+    public class When_message_is_moved_to_error_queue_using_dtc : NServiceBusAcceptanceTest
```

Comment:
@timbussmann I think that there are two problems here.  We do want to test that no side-effects are commited when doing SLR/Errors. This includes:
   *  User code operations enlisted in `TransactionScope` - this test does that
   *  Persister operations when `TransportTransaction` is shared between transport and persister. This would be tricky to test I think and we do not have tests for that.